lib/components/UploadPicker.vue

```diff
@@ -294,16 +294,27 @@ export default defineComponent({
 		},
 
 		hasFailure(): boolean {
-			return this.queue?.filter((upload: Upload) => upload.status === UploadStatus.FAILED).length !== 0
+			return this.queue?.some((upload: Upload) => upload.status === UploadStatus.FAILED)
 		},
```

Comment:
Wrong type here it is not boolean as there is a `?` operator. To make it boolean:

```ts
		hasFailure(): boolean {
			return this.queue && this.queue.some((upload: Upload) => upload.status === UploadStatus.FAILED)
		},
```

Reply:
But also the `queue` computed has then the wrong type.
Currently it is typed as `Upload[]` but then the `?` operator is not needed. So either the type is wrong or the operator.

Reply:
Seems like the `?` is not needed, it is always at least an empty array.